packages/api/src/modules/users/router.ts

```diff
@@ -133,4 +144,21 @@ router.post(
     }
 );
 
+router.post(
```

Comment:
```suggestion
router.patch(
```

Use patch, not post, you are updating data.